modules/ROOT/pages/functions/faq.adoc

```diff
@@ -0,0 +1,258 @@
+Serverless Functions FAQ
+========================
+
+Serverless Functions frequently asked questions.
+
+
+== What are Serverless Functions?
+
+Serverless Functions are a collection of tools that enable developers to create
+and run functions as a Knative Service on Kubernetes. The major components of
+this project are listed here.
+
+* Function runtimes for Go, Node.js and Quarkus
+* Buildpacks for Go, Node.js and Quarkus
+* A CLI for creating and managing functions as a Knative Service
```

Comment:
Please use lowercase "service" throughout because it'll make moving this to OCP docs easier